LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
```

Comment:
- [ ] It would be better if we fix our failing tests as seen in the screenshot below
![Screenshot from 2022-10-12 17-59-54](https://user-images.githubusercontent.com/46388113/195415431-a91ef047-e0e8-40b2-9d8b-5b5c24b0b0cc.png)